index.dd

```diff
@@ -200,6 +200,31 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Tiny RPN calculator,
+$(RUNNABLE_EXAMPLE_STDIN 2 3 3 4 + * *)
+----
+void main()
+{
+    import std.stdio, std.string,
+        std.algorithm, std.conv, std.meta;
+
+    // Reduce the RPN expression using a stack
+    readln.split.fold!((stack, op)
```

Comment:
Q1: Why do we need `readln` here?
Putting the string directly has the following advantages:
- People don't need to switch tabs to browse stdin
- We can take advantage of the fast, cached run.dlang.io backend

Q2: Why not the non-allocating `splitter`?